Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -109,40 +105,11 @@ export function install(args: Arguments) : Result {
             credentialProviderArguments = credentialProviderArguments.merge(nugetCredentialProviderArguments);
         }
 
-        if (Environment.hasVariable("QAUTHMATERIALROOT")) {
-            const qAuthMaterialRoot = Environment.getDirectoryValue("QAUTHMATERIALROOT");
-
-            const qAuthMaterial = {
-                environmentVariables: [{name: "QAUTHMATERIALROOT", value: qAuthMaterialRoot.path}],
-                unsafe: {
-                    untrackedScopes: [
-                        qAuthMaterialRoot
-                    ],
-                },
-            };
-
-            credentialProviderArguments = credentialProviderArguments.merge(qAuthMaterial);
-        }
-
         credentialProviderArguments = credentialProviderArguments.merge({
-            unsafe: {
-                untrackedPaths: [
-                    f`d:/app/autopilot.ini`,
-                ],
+            unsafe: {                
                 untrackedScopes: [
-                    d`d:/data/AutoPilotData`,
-                    d`d:/data/logs/AuthHelpers`,
-                    d`d:/data/Q/AuthHelpers`,
-                    d`d:/data/Q/QSecretsDPAPI`,
-                    d`d:/data/Q/RegionConfig`,
-                    d`d:/data/Q/TelemetryConfig`,
                     d`${Context.getMount("ProgramData").path}/Microsoft/Crypto`,

```

Comment:
>xt.getMount("ProgramData").path}/Micros [](start = 29, length = 39)

Perhaps if this one is still needed, it can be set as part of the nguetcrednetialproviderargumetns above?

Reply:
merged